picard/ui/matchdetailsdialog/matchdetails.py

```diff
@@ -0,0 +1,226 @@
+                self.set_table_item_val(display_row_nbr, 'trk_id', entry.trk_id)
+                self.set_table_item_val(display_row_nbr, 'trk_pos', entry.trk_pos)
+                self.set_table_item_val(display_row_nbr, 'med_trk_cnt', entry.med_trk_cnt)
+                self.set_table_item_val(display_row_nbr, 'rel_tot_trk_cnt', entry.rel_tot_trk_cnt)
+                self.set_table_item_val(display_row_nbr, 'parts', str(entry.parts))
+                self.set_table_item_val(display_row_nbr, 'release_parts', str(entry.release_parts))
+                self.set_table_item_val(display_row_nbr, 'recording_base_score', entry.recording_base_score)
+
+        self.show_table(selected_row_nbr=selected_row_nbr, sort_column='similarity', sort_order=QtCore.Qt.DescendingOrder)
+
+    def accept_event(self, selected_rows_user_values):
+        if not selected_rows_user_values:
+            return
+
+        selected_entry_nbr = selected_rows_user_values[0]
+        match_details_log.set_file_result_selected_entry_nbr(self.current_file.filename, selected_entry_nbr)
+        self.load_selection(selected_entry_nbr)
+
+    def load_selection(self, entry_nbr):
+        """Load the album corresponding to the selected track.
+
+        Allow user to pick from our new screen and use that track instead of one its currently assigned to.
+        Based on logic from searchdialog/track.py line 186
+        """
+
+        entry = self.entries[entry_nbr]
+
+        if entry.rel_id:
+            # The track is not an NAT
+            self.tagger.get_release_group_by_id(entry.rg_id).loaded_albums.add(entry.rel_id)
+
+            # Have to move that file from its existing album to the new one.
+            if isinstance(self.current_file.parent, Track):
+                album = self.current_file.parent.album
+                self.tagger.move_file_to_track(self.current_file, entry.rel_id, entry.rec_id)
+                if not album._files:
+                    # Remove album if it has no more files associated
+                    self.tagger.remove_album(album)
+            else:
+                self.tagger.move_file_to_track(self.current_file, entry.rel_id, entry.rec_id)
+        else:
+            if self.current_file and getattr(self.current_file.parent, 'album', None):
+                album = self.current_file.parent.album
+                self.tagger.move_file_to_nat(self.current_file, entry.rec_id, None)
+                if album._files == 0:
+                    self.tagger.remove_album(album)
+            else:
+                self.tagger.load_nat(entry.rec_id, None)
+                self.tagger.move_file_to_nat(self.current_file, entry.rec_id, None)
+
+        log.debug("using alternative release id:" + entry.rel_id)
+
+    def export_match_dtl_entries_to_file(self):
+        """ Export entries to a file that can be viewed in external program
+        """
+        try:
+            # Prompt user for file location to save to
+            dir = str(Path.home())
```

Comment:
`dir` is a reserved keyword, and in this case rather useless intermediate variable.
This should work as well:

```python
os.path.join(Path.home(), "picard_match_results.tsv")
```